cli/src/cli.rs

```diff
@@ -1070,9 +1084,12 @@ fn process_create_address_with_seed(
     seed: &str,
     program_id: &Pubkey,
 ) -> ProcessResult {
-    let config_pubkey = config.pubkey()?;
-    let from_pubkey = from_pubkey.unwrap_or(&config_pubkey);
-    let address = create_address_with_seed(from_pubkey, seed, program_id)?;
+    let from_pubkey = if let Some(pubkey) = from_pubkey {
```

Comment:
NIT: `unwrap_or()`

Reply:
Can't use `?` in a closure that returns a Pubkey, so the current construction seems simpler to me for the `else` case.